### PR TITLE
checkout if go-get is successful

### PR DESCRIPTION
**Pull request by knightXun**
sometimes go get may fail.we should check it.

### CODE REVIEW

File: bin/ci2gubernator.sh [hunk 1]
```diff
@@ -29,6 +29,11 @@ openssl aes-256-cbc -d -in "${ENCRYPTED_SA_JSON}" -out "${TMP_SA_JSON}" -k "${GC
 
 go get -u istio.io/test-infra/toolbox/ci2gubernator
 
+if [[ -d "/go/bin/ci2gubernator" ]];then
```
Comment (sdake) [inline bin/ci2gubernator.sh]: not sure why /go/bin/cigubernator would be the target of a go get operation.  Entirely depends on environment variables.

File: bin/codecov.sh [hunk 2]
```diff
@@ -118,6 +118,12 @@ fi
 
 echo 'Checking package coverage'
 go get -u istio.io/test-infra/toolbox/pkg_check
+
+if [[ -d "/go/bin/pkg_check" ]];then
```
Comment (sdake) [inline bin/codecov.sh]: same issue

File: bin/ci2gubernator.sh (hunk 1 above)
Comment (knightXun) [reply bin/ci2gubernator.sh]: ok,i go to fix it.

**Comment by knightXun** [comment]
ping @sdake, this pr seems ok.

**Pull request merged**